runelite-client/src/main/java/net/runelite/client/plugins/music/MusicPlugin.java

```diff
@@ -588,7 +593,11 @@ public void update()
 		public void shutDown()
 		{
 			super.shutDown();
-			handle.setSpriteId(SpriteID.SETTINGS_SLIDER_HANDLE_BLUE);
+
+			if (this.handle != null)
```

Comment:
I don't see any errors relating to the handle. What is this updated for?

Reply:
Noticed in my own logs, happens when u toggle "Granular volume sliders" on the login screen.